tests/functional/generate_pages.py

```diff
@@ -60,7 +60,7 @@
     browser.openQuestionnaire(schema);
   });
 
-  it("Given..., When..., Then...", () => {
+  it("Given..., When..., Then I see...", () => {
```

Comment:
Bad replace.

Reply:
What should this have been? Unedited?

Reply:
Ya, the `Then I see` was for your test example.

Reply:
Ah sorry, I misinterpreted the prior comment to mean for all of them - reverted 👍